### PR TITLE
[EDU-3686] refactor: change custom ed programs to instructor-led

### PR DESCRIPTION
**Pull request by bru-andrade**
### Changes

- Modify custom ed program files

### Additional links

[EDU-3000]: https://aziontech.atlassian.net/browse/EDU-3000?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[EDU-3686]: https://aziontech.atlassian.net/browse/EDU-3686?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/main-menu/reference/services/integration-services/instructor-led-training.mdx [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 
-**Custom Education Programs** offer personalized learning experiences tailored to your specific needs. Expand your knowledge and skills of our platform and products with expert-led training in-person or virtual classes.
+**Instructor-Led Training** offers learning experiences adapted to your specific needs. Expand your knowledge and skills of our platform and products with expert-led training in-person or virtual classes.
 
 If you have an **Enterprise** or **Mission Critical** Service Plan, you can request this service by contacting the [Sales team](https://www.azion.com/en/contact-sales/).
 
 For more detailed information, see [Azion Service Plans](https://www.azion.com/en/pricing/#plans).
 
-In addition to the Custom Education Programs, Azion provides full access to comprehensive documentation through:
+In addition to the Azion Instructor-Led Training, Azion provides full access to comprehensive documentation through:
```
Comment (bru-andrade) [inline src/content/docs/en/pages/main-menu/reference/services/integration-services/instructor-led-training.mdx:15]:
```suggestion
In addition to Instructor-Led Training, Azion provides full access to comprehensive documentation through:
```

File: src/content/docs/en/pages/main-menu/reference/services/integration-services/instructor-led-training.mdx [hunk 2]
```diff
@@ -1,18 +1,18 @@
 ---
-title: Custom Education Programs
-description: It offers personalized learning experiences tailored to your specific needs. Expand your knowledge and skills of Azion's platform and products with expert-led training in-person or virtual classes.
-meta_tags: services, integration, custom education programs
+title: Instructor-Led Training
+description: Instructor-Led Training offers learning experiences adapted to your specific needs. Expand your knowledge and skills of our platform and products with expert-led training in-person or virtual classes.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/services/integration-services/instructor-led-training.mdx:3]:
```suggestion
description: Instructor-Led Training offers learning experiences adapted to your specific needs.
```

File: src/content/docs/en/pages/main-menu/reference/services/integration-services/instructor-led-training.mdx [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
+meta_tags: services, integration, custom education programs, coach, teach, learn
 namespace: documentation_services_custom_education_programs
 permalink: /documentation/services/custom-education-programs/
 ---
 
-**Custom Education Programs** offer personalized learning experiences tailored to your specific needs. Expand your knowledge and skills of our platform and products with expert-led training in-person or virtual classes.
+**Instructor-Led Training** offers learning experiences adapted to your specific needs. Expand your knowledge and skills of our platform and products with expert-led training in-person or virtual classes.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/services/integration-services/instructor-led-training.mdx:9]:
```suggestion
**Instructor-Led Training** offers learning experiences adapted to your specific needs. Expand your knowledge and skills of Azion's platform and products with expert-led training in-person or virtual classes.
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/servicos/instructor-led-training.mdx [hunk 4]
```diff
@@ -1,24 +1,22 @@
 ---
-title: Custom Education Programs
+title: Instructor-Led Training
 description: Custom Education Programs oferecem experiências de aprendizagem personalizadas adaptadas às suas necessidades específicas.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/servicos/instructor-led-training.mdx:3]:
```suggestion
description: Instructor-Led Training oferece experiências de aprendizagem personalizadas adaptadas às suas necessidades específicas.
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/servicos/instructor-led-training.mdx [hunk 5]
```diff
@@ -1,24 +1,22 @@
...
-meta_tags: services, integration, custom education programs
+meta_tags: services, integration, custom education programs, instructor, instructor-led, coach, teach, learn
 namespace: documentation_services_custom_education_programs
 permalink: /documentacao/servicos/custom-education-programs/
 ---
 
-Os **Custom Education Programs** oferecem experiências de aprendizagem adaptadas às suas necessidades específicas. Expanda seu conhecimento e habilidades da plataforma e dos produtos Azion com treinamentos presenciais ou virtuais conduzidos por especialistas.
+**Instructor-Led Training** oferece experiências de aprendizado adaptadas às suas necessidades específicas. Expanda seu conhecimento e habilidades da plataforma e dos produtos Azion com treinamentos presenciais ou virtuais conduzidos por especialistas.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/servicos/instructor-led-training.mdx:9]:
```suggestion
**Instructor-Led Training** oferece experiências de aprendizado adaptadas às suas necessidades específicas. Expanda seu conhecimento e habilidades da plataforma e dos produtos da Azion com treinamentos presenciais ou virtuais conduzidos por especialistas.
```

**Review by MarianaAguilera — APPROVED** [review]

**Review by hannah-sk — APPROVED** [review]
com comments da @MarianaAguilera

**Pull request merged**